include/asterisk/stasis_channels.h

```diff
@@ -656,6 +656,14 @@ struct stasis_message_type *ast_channel_talking_start(void);
  */
 struct stasis_message_type *ast_channel_talking_stop(void);
 
+/*!
+ * \since 21.3.2
```

Comment:
This is no longer correct, and this change is not commented to be cherry picked against 21 anyway

Reply:
`\since 22` now, is that ok?